apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -49,36 +49,39 @@ type KeyPair = {
 	private_key: string | null;
 };
 
+const E2EEStateDependency = new Tracker.Dependency();
```

Comment:
This tracker dependency, could you point me out which tracker.autorun needs it? I'm having a hard time finding it...